src/peer-connection.ts

```diff
@@ -215,6 +229,12 @@ class PeerConnection extends EventEmitter<PeerConnectionEventHandlers> {
         });
     }
 
+    this.emit(PeerConnection.Events.SdpChange, {
+      type: 'offer',
+      munged: true,
+      sdp: description?.sdp,
+    });
+
     return this.pc.setLocalDescription(description);
```

Comment:
It's possible for the call to `this.pc.setLocalDescription` to fail, in which case we wouldn't want to emit an event saying the SDP changed. We probably only want to emit this event after `this.pc.setLocalDescription` succeeds.